### PR TITLE
Tracking PR for release `0.18.0`

### PR DESCRIPTION
**Pull request by tcharding**
I'm not sure what we want to consider for merge before cutting the next release but at least we need:

- #312 
- #311

### CODE REVIEW

**Comment by tcharding** [comment]
Other possible PRs that look ready/interesting for next release:
- #295

**Comment by tcharding** [comment]
I messaged @stevenroose on another platform but for transparency I'd like to know what's the status of this crate? Are you watching this repo @apoelstra? In order to upgrade `rust-miniscript` we need to upgrade `bitcoind` which needs an upgrade of this crate. cc @RCasatta

**Comment by RCasatta** [comment]
Yeah, to upgrade bitcoind I need an upgrade here, I just merged #312 to move things forwards, but we will need steven to publish.

**Review by RCasatta — APPROVED** [review]
ACK f0efe223df7b69b380e44ce2cf82a2486c67c9b4

File: integration_test/src/main.rs [hunk 1]
```diff
@@ -1367,18 +1367,22 @@ fn test_add_multisig_address(cl: &Client) {
     assert!(cl.add_multisig_address(addresses.len(), &addresses, None, Some(json::AddressType::Bech32)).is_ok());
 }
 
+#[rustfmt::skip]
```
Comment (stevenroose) [inline integration_test/src/main.rs:1370]: :sweat_smile:

**Review by stevenroose — APPROVED** [review]
ACK f0efe223df7b69b380e44ce2cf82a2486c67c9b4

**Pull request merged**